core/utils.py

```diff
@@ -415,7 +415,7 @@ def set_url_query_parameter(
     query_params[param_name] = [param_value]
     new_query_string = urllib.parse.urlencode(query_params, doseq=True)
 
-    return python_utils.url_unsplit( # type: ignore[no-any-return, no-untyped-call]
+    return urllib.parse.urlunsplit( # type: ignore[no-any-return, no-untyped-call]
```

Comment:
Either this
```suggestion
    return urllib.parse.urlunsplit( # type: ignore[no-any-return]
```
or this 
```suggestion
    return urllib.parse.urlunsplit(
```
should fix one of the MyPy errors.